contrib/helm-charts/portus/templates/portus-deployment.yaml

```diff
@@ -20,121 +20,153 @@ spec:
         release: {{ .Release.Name }}
     spec:
       containers:
-        - name: {{ template "portus.fullname" . }} 
```

Comment:
Can you double check that moving to a static name like "portus" for the container here doesn't violate any of the helm chart requirements?